_drafts/2021-02-08-custom_markdown_for_github_pages.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: "Making Custom Markdown for Github Pages"
+description: >
+  I love Markdown, I take all my notes in it and write my blog in it. But
+  sometimes you want to create new syntax; read on to find out how!
+image: /files/jekyll/stansaab_censor_and_operator.jpg 
+image_alt: >
+  A black and white photo of a man in a short-sleeve collared shirt uses an
+  osciliscope to adjust a circuit board on a Stansaab 'Censor' computer at
+  Standard Radio's factory in Ulvsunda, Stockholm.
+categories:
+  - software-development
+---
+
+{% capture file_dir %}/files/jekyll/{% endcapture %}
+
+[Markdown] is a great way to write; simple enough to be read as text, with all
+the power of HTML. I have been using it for my own notes for a decade and this
```

Comment:
"all the power of HTML" isn't really true, but also that's sorta the point? HTML can do anything, but Markdown only does the stuff you care about

Reply:
It actually is true! Markdown is a strict superset of HTML. 😊

But I think I get your point.